messages/training-checkout.txt

```diff
@@ -6,17 +6,31 @@ Thanks again for taking part in Software Carpentry (SWC) and Data Carpentry (DC)
+We strongly prefer contributions to the Data Carpentry lessons right now (since their material is much younger), or to our fledgling "Python for complete novices" lesson.  You can find links to the Data Carpentry lessons on http://www.datacarpentry.org/lessons/, and the Python lesson on https://github.com/swcarpentry/python-novice-gapminder.  Whatever you contribute to, and however you do it (by pull request or by adding to the Google Doc), please:
 
-3. Once you're through the discussion session(s), sign up for a 5-minute online demonstration lesson slot on http://pad.software-carpentry.org/teaching-demos (or add yourself to the top of the pad if none of the available slots works for you).  In that, we will ask you to teach a short segment from your chosen lesson - we'll pick the starting point on the day, so you will need to be familiar with the whole lesson.  If that goes well, we will send you your certificate, add you to our roster, and get you on the dance floor.
+a) Look at how the existing exercises in that lesson are formatted, and format yours in the same way.
 
-Your change can be a fix to the existing material, a new exercise, or a diagram of some kind (preferably in a vector format like SVG, but we'll take anything).  We are *not* presently accepting material with new concepts: the lessons are full to bursting as it is, and our rule is that new stuff can only be added if we can identify something to take out.  (Note: refactorings that rearrange the order of material are also welcome.)
+b) If you're submitting by PR, please include discussion in the comments on the PR (*not* in the submitted content) about the purpose of the exercise.
 
-Your change doesn't have to be accepted in order for you to become an instructor - the point is to make sure that you understand the process and have met the people who review material.  If you want to do your final demonstration(s) in a language other than English, please let us know in advance so that we can find someone to sit in.
+Please note:
+
+- We do *not* want exercises or other material that introduce new concepts - our lessons are already full to bursting.
+
+- We accept submissions to DC for the purpose of becoming a SWC instructor.
+
+- Your change doesn't have to be accepted in order for you to become an instructor - the point is to make sure that you understand the process and have met the people who review material.
+
+2. Once you have submitted, pick a SWC lesson (if you want to become a SWC instructor) and/or a DC lesson (if you want to teach DC - if you want to teach both, pick one of each) and read through it carefully.  Once you've done that, sign up on http://pad.software-carpentry.org/lesson-discussion-2016 to take part in a discussion session.  Some of these sessions focus on specific lessons, while others are combined with our workshop debriefing sessions.  We're trying both formats for the next month or so to figure out which works best.
```

Comment:
Suggest different wording and order of saying that. I.e. tell them to pick lesson(s) so that they read through them, submit a PR and join a discussion session. Spell out that they need to be active at the discussion session and not just listen!